source/material_model/rheology/composite_visco_plastic.cc

```diff
@@ -194,7 +197,8 @@ namespace aspect
                                                                    diffusion_creep_parameters,
                                                                    dislocation_creep_parameters,
                                                                    peierls_creep_parameters,
-                                                                   drucker_prager_parameters);
+                                                                   phase_function_values,
```

Comment:
In line 170 above, drucker_prager_parameters is already computed. Could you use that here instead of passing the phase_function_values and n_phases_per_composition?